server/api/api.go

```diff
@@ -2208,6 +2209,62 @@ func (a *API) handleGetTemplates(w http.ResponseWriter, r *http.Request) {
 	auditRec.Success()
 }
 
+func (a *API) handleGetDefaultTemplates(w http.ResponseWriter, r *http.Request) {
```

Comment:
Not sure I understand the reasoning to make this a separate API versus fetching templates from team 0.

Reply:
Yeah, this is not needed. It was needed in the middle of the implementation but I changed something and now it's not needed anymore.

Removed the API.